datashuttle/utils/rclone.py

```diff
@@ -205,6 +202,15 @@ def prompt_rclone_download_if_does_not_exist() -> None:
         )
 
 
+def check_rclone_with_default_call() -> bool:
```

Comment:
This change just swaps the order of the functions so that the called function is under the calling function